spirv_cross/src/bindings.rs

```diff
@@ -1586,6 +1586,9 @@ pub mod root {
     pub struct ScMslCompilerOptions {
         pub vertex_transform_clip_space: bool,
         pub vertex_invert_y: bool,
+        pub version: u32,
```

Comment:
nit: could we change the version to an enum to match [GLSL](https://github.com/grovesNL/spirv_cross/blob/master/spirv_cross/src/glsl.rs#L17) and [HLSL](https://github.com/grovesNL/spirv_cross/blob/master/spirv_cross/src/hlsl.rs#L20)?